charts/runner/templates/deployment.yaml

```diff
@@ -16,6 +16,7 @@ spec:
       annotations:
         rollme: {{ randAlphaNum 5 | quote }}
     spec:
+      terminationGracePeriodSeconds: 3600
```

Comment:
Do you know if this will affect all ways of killing a pod? I.e., if we try to kill it from the CLI, will it take 1h to die?

Reply:
Do we have any idea about why the pods are killed, in the first place?

Reply:
Ah, yes this we already knew, thanks for the reminder. Do you know if during the grace period the pod can take additional jobs and reset the timer? Or would we have to wait 1h to be able to create a new pod for a new PR?

Reply:
If the POD is busy when it receives the terminate request, it won't accept any other job.